frontend/src/routes/Infrastructure/Clusters/ManagedClusters/components/NodePoolsTable.tsx

```diff
@@ -86,17 +86,20 @@ const NodePoolsTable = ({ nodePools, clusterImages }: NodePoolsTableProps): JSX.
         [t]
     )
 
-    const getAutoscaling = useCallback((nodepool: NodePool) => {
-        const autoscaling = get(nodepool, 'spec.autoScaling')
-        if (!autoscaling) {
-            return ''
-        }
+    const getAutoscaling = useCallback(
+        (nodepool: NodePool) => {
+            const autoscaling = get(nodepool, 'spec.autoScaling')
+            if (!autoscaling) {
+                return t('False')
+            }
 
-        const min = autoscaling.min || 1
-        const max = autoscaling.max || 1
+            const min = autoscaling.min || 1
+            const max = autoscaling.max || 1
 
-        return `Min ${min} Max ${max}`
-    }, [])
+            return `Min ${min} Max ${max}`
```

Comment:
This string needs to be externalized for translation.

Reply:
Done. Not sure if that was the best way to do it, let me know if there's a better way.

Reply:
Probably better to use interpolation, like `t('Min {{min}} Max {{max}}', {min, max})`. When you commit, it should automatically add to translations.json and remove the old ones.

Reply:
Thanks, I tried something similar but couldn't get it to work